### ISSUE TITLE
Pulmonary Valve Regurgitation

### ISSUE THREAD

**Issue by taylordm** [obophenotype/human-phenotype-ontology#7190]
**Preferred term label:**
Pulmonary Valve Regurgitation

**Synonyms**
pulmonic regurgitation

**Definition (free text, please give PubMed ID)**
Pulmonary regurgitation (PR, also called pulmonic regurgitation) is a leaky pulmonary valve. This valve helps control the flow of blood passing from the right ventricle to the lungs. A leaky pulmonary valve allows blood to flow back into the heart chamber before it gets to the lungs for oxygen.

https://www.heart.org/en/health-topics/heart-valve-problems-and-disease/heart-valve-problems-and-causes/problem-pulmonary-valve-regurgitation

**Parent term (use hpo.jax.org/app)**
Pulmonary valve defects HP:0005148

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM:  600001 HEART DEFECTS, CONGENITAL, AND OTHER CONGENITAL ANOMALIES; HDCA

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-3302-4610

**Comment by pnrobinson** [comment]
This is a synonym for Pulmonary insufficiency HP:0010444. I will update the definition and synonym list (there are other synonyms too).

**Comment by pnrobinson** [comment]
I added these synonyms, thanks!
